tests/avifyuv.c

```diff
@@ -221,8 +221,13 @@ int main(int argc, char * argv[])
                                 }
                             }
 
-                            avifImageRGBToYUV(image, &srcRGB);
-                            avifImageYUVToRGB(image, &dstRGB);
+                            if (avifImageRGBToYUV(image, &srcRGB) != AVIF_RESULT_OK || avifImageYUVToRGB(image, &dstRGB) != AVIF_RESULT_OK) {
+                                avifRGBImageFreePixels(&srcRGB);
```

Comment:
How about a goto Error ?

Reply:
It might make sense here but I'm not a big fan of goto and I feel like this is an old test that's not worth spending more time on...

Reply:
totally right, please go ahead